ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/hosts.yml

```diff
@@ -205,6 +205,21 @@ all:
               hosts:
                 ptp-tests-leaf1:
                 ptp-tests-leaf2:
+        SFLOW_TESTS:
```

Comment:
please add a test of one l2leaf as well, so we test the port-channel uplink. If you add it before fixing my other comment, you will see why we need to move the definition. (You will see sflow on the port-channel member).

Reply:
done! Makes sense now!